euth/contrib/static/js/app.js

```diff
@@ -15,3 +15,23 @@ module.exports = {
   'renderFollow': ReactFollow.renderFollow,
   'renderUserList': ReactUserList.renderUserList
 }
+
+var initilizeWidget = function (project, name, initializer) {
+  if (!initializer) {
+    initializer = name
+    name = project
+    project = 'euth'
+  }
+
+  document.addEventListener('DOMContentLoaded', function () {
```

Comment:
I am not entirely sure of the differences, but `$(document).ready()` could be more robust. A common case is that it will still execute if the content is already loaded.